src/test/java/emissary/test/core/junit5/UnitTest.java

```diff
@@ -42,7 +42,13 @@ public abstract class UnitTest {
     // Runtime typed logger
     protected Logger logger = LoggerFactory.getLogger(this.getClass());
 
-    protected static List<String> answerFiles = new ArrayList<>();
+    /**
+     * Alternative answer files to use if loading answer files from a non-default location. This list should be populated
+     * using the {@link #useAlternateAnswerFileSource(Class)} method.
+     */
+    private final List<String> answerFiles = new ArrayList<>();
+
+    protected final AtomicReference<Class<?>> answerFileClassRef = new AtomicReference<>(getClass());
```

Comment:
This is here only to facilitate potential access by RegressionTest in follow-on work. Could be cleaner approaches.